src/styles/utils/index.ts

```diff
@@ -847,15 +846,14 @@ function displayIfTrue(condition: boolean): ViewStyle {
 /**
  * Gets the correct height for emoji picker list based on screen dimensions
  */
-function getEmojiPickerListHeight(hasAdditionalSpace: boolean, windowHeight: number): ViewStyle {
+function getEmojiPickerListHeight(isRenderingShortcutRow: boolean, windowHeight: number): ViewStyle {
```

Comment:
This function appears unused now:

<img width="1401" alt="image" src="https://github.com/Expensify/App/assets/47436092/bb6018cd-4caa-4072-b159-d38345957e6b">

let's delete it from this file accordingly